src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug

```diff
@@ -7,24 +7,30 @@
+          .fg-line(ng-if='!ctrl.full24HoursDay')
+            time-select(
+              date="ctrl.event.start",
+              on-time-change="ctrl.onStartDateChange",
+              locale="{{ ::ctrl.locale }}",
+              time-format="{{ ::ctrl.timeFormat }}"
+            )
       .col-xs-12.col-md-4
         .form-group
           .fg-line
-            input.input.form-control.date(type="text", size="10", class="form-control", ng-model="ctrl.event.end", data-autoclose="1", data-start-week="1", data-date-format="{{::ctrl.dateFormat}}", data-min-date="{{ctrl.getMinDate()}}", placeholder="{{ 'End date' | translate }}", ng-change="ctrl.onEndDateChange()", aria-expanded="false", ng-blur="ctrl.dateOnBlurFn()", bs-datepicker, cal-date-to-moment, cal-friendlify-end-date, data-is-all-day="ctrl.allDay")
-          .fg-line(ng-if='!ctrl.allDay')
-            input.input(type="text", size="10", class="form-control", ng-model="ctrl.event.end", data-autoclose="0", placeholder="{{ 'Time' | translate }}", ng-change="ctrl.onEndDateChange()", aria-expanded="false", ng-disabled="ctrl.allDay", ng-blur="ctrl.dateOnBlurFn()", lng-clockpicker, lng-clockpicker-options="{twelvehour: !ctrl.use24hourFormat}")
-          .fg-line(ng-if='ctrl.allDay')
-            input.input(type="text", size="10", class="form-control", ng-model="ctrl.previousEnd", data-autoclose="0", placeholder="{{ 'Time' | translate }}", ng-change="ctrl.onEndDateChange()", aria-expanded="false", ng-disabled="ctrl.allDay", ng-blur="ctrl.dateOnBlurFn()", lng-clockpicker, lng-clockpicker-options="{twelvehour: !ctrl.use24hourFormat}")
+            input.input.form-control.date(type="text", size="10", class="form-control", ng-model="ctrl.event.end", data-autoclose="1", data-start-week="1", data-date-format="{{::ctrl.dateFormat}}", data-min-date="{{ctrl.getMinDate()}}", placeholder="{{ 'End date' | translate }}", ng-change="ctrl.onEndDateChange()", aria-expanded="false", ng-blur="ctrl.dateOnBlurFn()", bs-datepicker, cal-date-to-moment, cal-friendlify-end-date, data-is-all-day="ctrl.full24HoursDay")
+          .fg-line(ng-if='!ctrl.full24HoursDay')
+            time-select(
+              date="ctrl.event.end",
+              on-time-change="ctrl.onEndDateChange",
+              locale="{{ ::ctrl.locale }}",
+              time-format="{{ ::ctrl.timeFormat }}"
+            )
       .col-xs-12.col-md-4
         .form-group.nopadding
           .fg-line
             .checkbox.allday
               label
-                input(type="checkbox", ng-model='ctrl.allDay', ng-change="ctrl.setEventDates();")
+                input(type="checkbox", ng-model='ctrl.full24HoursDay', ng-change="ctrl.setEventDates();")
```

Comment:
This fix isn't related to this issue, right?

Reply:
well to some extent it is, the time select does not behave correctly when using `allDay` ( which doesn't exist in the controller ).

the logic was not changed, I used `full24HoursDay` to judge whether the user wants to suggest another date ( the all-day tick is checked ) if not the time select appears.

Reply:
You should have a separate commit for this then.

Reply:
alright